eventstore/models.py

```diff
@@ -160,3 +160,27 @@ class PostbirthRegistration(models.Model):
     timestamp = models.DateTimeField(default=timezone.now)
     created_by = models.CharField(max_length=255, blank=True, default="")
     data = JSONField(default=dict, blank=True, null=True)
+
+
+class Messages(models.Model):
+    INBOUND = "I"
+    OUTBOUND = "O"
+    DIRECTION_TYPES = [(INBOUND, "Inbound"), (OUTBOUND, "Outbound")]
+
+    id = models.CharField(max_length=255, primary_key=True, blank=True)
+    contact_id = models.CharField(max_length=255, blank=True)
+    timestamp = models.DateTimeField(default=timezone.now)
+    type = models.CharField(max_length=255, blank=True)
+    data = JSONField(default=dict, blank=True, null=True)
+    message_direction = models.CharField(max_length=3, choices=DIRECTION_TYPES)
```

Comment:
None of our types have a length longer than 1, so we can reduce the `max_length` here to 1

Reply:
okay will do